controller/controller_server/main.py

```diff
@@ -0,0 +1,64 @@
+import time
+from argparse import ArgumentParser
+from concurrent import futures
+
+import grpc
+
+from controller.common import settings
+from controller.common.csi_logger import get_stdout_logger, set_log_level
+from controller.controller_server.csi_controller_server import CSIControllerServicer
+from controller.csi_general import csi_pb2_grpc
+
+logger = get_stdout_logger()
+
+
+class ControllerServicer(CSIControllerServicer):
```

Comment:
I don't think the main file should contain this class.
In addition, I don't think this class should inherit from the big servicer class.

Reply:
then what do you suggest?
I was thinking of changing this to `ControllerServicer(CSIControllerServicer, ReplicationControllerServicer)` in the upcoming PR

Reply:
I think we should have a design review session with @ArbelNathan before doing so.

Reply:
do you have any suggestion to prepare for the session?

Reply:
The session is done. We agreed to have a composition design instead of inheritance.

Reply:
done

Reply:
Actually, it's not (fully) done. I still think we should have the class in a different file.

Reply:
and what would be the name of said file?

Reply:
path/to/folder/class_name.py

Since this class isn't a controller servicer, but it's controller servicer manager/owner/initializator/starter (other suggestion?) I suggest renaming it accordingly.

Reply:
done